science_node/src/science_controller.cpp

```diff
@@ -1,6 +1,6 @@
 
 #include "ros/ros.h"
-#include "std_msgs/Float32.h"
+#include "std_msgs/Float32MultiArray.h"
```

Comment:
Curious: any reason u made this change?)

Reply:
the way xbox is written makes it easy to send commands this way. i also thought it was cleaner so I didnt have to have 4 publishers